wallet/src/main/java/com/android/identity_credential/wallet/ui/destination/main/MainScreen.kt

```diff
@@ -159,10 +161,22 @@ fun MainScreenContent(
     cardViewModel: CardViewModel,
     scope: CoroutineScope,
     drawerState: DrawerState,
-    permissionTracker: PermissionTracker
+    permissionTracker: PermissionTracker,
+    capabilities: AndroidKeystoreSecureArea.Capabilities,
 ) {
     val snackbarHostState = remember { SnackbarHostState() }
     val haveRequiredPermissions = permissionTracker.granted(blePermissions)
+    var showLskfWarning by remember { mutableStateOf(false) }
```

Comment:
Nit: I would call `showDeviceLockNotSetupWarning`... yes, it's longer but also more descriptive!